gutenberg.php

```diff
@@ -527,3 +527,11 @@ function toggleDropdown() {
 function gutenberg_add_admin_body_class( $classes ) {
 	return "$classes gutenberg-editor-page";
 }
+
+if (
+	defined( 'GUTENBERG_DEVELOPMENT_MODE' ) &&
+	GUTENBERG_DEVELOPMENT_MODE &&
+	isset( $_GET['e2e_test_plugin'] )
+) {
+	require_once 'test/e2e/test-plugins/' . $_GET['e2e_test_plugin'] . '.php';
```

Comment:
Not sure how I feel about that, I feel like this shouldn't be included in the php it self, it's more related to the test environment setup. Also, it's not the only plugin we'd want to test, this is just one of the several extensibility patterns we'd want to test.

I think this is 👎 for me :(

Reply:
It allows for more than one plugin to be tested. Sure it's related to the test environment, but it will only be included where necessary. I'm not sure what the difference here is between including it in PHP and having a plugin permanently visible...

Alternatively we can move the line that loads the plugin folder from the Docker config to someone PHP, but not sure if that's much better.

Reply:
I don't understand the need to test activating/deactivating plugins. Anyway, reverted the change.